src/rapidata/rapidata_client/validation/validation_set_manager.py

```diff
@@ -399,22 +411,24 @@ def create_timestamp_set(self,
                 )
             )
 
-        return self._submit(name=name, rapids=rapids, print_confirmation=print_confirmation)
+        return self._submit(name=name, rapids=rapids, dimensions=dimensions, print_confirmation=print_confirmation)
     
     def create_mixed_set(self,
         name: str,
         rapids: list[Rapid],
+        dimensions: list[str] = ["global"],
```

Comment:
I think similar story to the one above, let's let the backend dictate what a default value is and only update it if we explicitly set dimensions. This way we could change the default to something else without any old SDK implementations interferring